mkosi

```diff
@@ -129,7 +130,13 @@ def create_image(args, workspace):
     print_step("Creating partition table...")
 
     f = tempfile.NamedTemporaryFile(dir = os.path.dirname(args.output), prefix='.mkosi-')
-    subprocess.run(["chattr", "+C", f.name], stdout=subprocess.DEVNULL, stderr=subprocess.DEVNULL, check=True)
+    try:
+        subprocess.run(["chattr", "+C", f.name], stdout=subprocess.PIPE, stderr=subprocess.PIPE, check=True)
+    except subprocess.CalledProcessError as e:
+        if e.stderr.startswith(("chattr: " + os.strerror(errno.ENOTTY)).encode()) or e.stderr.startswith(("chattr: " + os.strerror(errno.ENOTSUP)).encode()):
+            pass # file system does not support "no copy on write" attribute
+        else:
+            raise e
```

Comment:
I guess that wouldn’t hurt either… should a warning be printed to the console, or the error be ignored completely silently?